src/status_im/ui/screens/wallet_connect/session_proposal/views.cljs

```diff
@@ -0,0 +1,215 @@
+(ns status-im.ui.screens.wallet-connect.session-proposal.views
+  (:require-macros [status-im.utils.views :refer [defview letsubs]])
+  (:require [re-frame.core :as re-frame]
+            [status-im.ui.components.react :as react]
+            [status-im.i18n.i18n :as i18n]
+            [status-im.utils.security]
+            [quo.design-system.colors :as colors]
+            [quo.core :as quo]
+            [status-im.ui.components.icons.icons :as icons]
+            [status-im.ui.components.bottom-panel.views :as bottom-panel]
+            [status-im.ui.screens.wallet-connect.session-proposal.styles :as styles]
+            [status-im.ui.components.list.views :as list]
+            [reagent.core :as reagent]
+            [clojure.string :as string]
+            [quo.platform :as platform]))
+
+(def chevron-icon-container-width 24)
+
+(def chevron-icon-container-height 24)
+
+(defn toolbar-selection [{:keys [text background-color on-press]}]
+  [react/touchable-opacity {:on-press on-press}
+   [react/view (styles/toolbar-container background-color)
+    [quo/text {:color :inverse
+               :weight :medium
+               :style styles/toolbar-text}
+     text]
+    [icons/icon
+     :main-icons/chevron-down
+     {:color  (:text-05 @colors/theme)
+      :width  chevron-icon-container-width
+      :height chevron-icon-container-height}]]])
+
+(def show-account-selector? (reagent/atom false))
+
+(defn render-account [{:keys [address name color] :as account} _ _ {:keys [selected-account on-select]}]
+  (let [account-selected? (= (:address @selected-account) address)]
+    [react/touchable-without-feedback {:on-press #(do
+                                                    (reset! selected-account (merge {} account))
+                                                    (when on-select (on-select)))}
+     [react/view (styles/account-container color account-selected?)
+      [quo/text {:color :inverse
+                 :weight (if account-selected? :medium :regular)}
+       name]]]))
+
+(defn account-selector [accounts selected-account on-select]
+  [react/view styles/account-selector-container
+   [react/view
```

Comment:
redundant [react/view

Reply:
Thanks, I will review the views hierarchy and see how to improve it